client/src/Filters.js

```diff
@@ -0,0 +1,151 @@
+// Nav bar
+// - logo on left, icon, hello on right, logout on right
+// - logo is static / does nothing
+// - logout - empty function on click, maybe  console.log('logged out')
+
+// list vs map buttons
+//
+
+// filtering sidebar on left
+// filter title, reset button which removes all selects
+// div box with scroll
+// -
+
+import './Filter.css';
+import React from 'react';
+
+import Select, { components } from 'react-select';
+
+function arrayMove(array, from, to) {
+  array = array.slice();
+  array.splice(to < 0 ? array.length + to : to, 0, array.splice(from, 1)[0]);
+  return array;
+}
+
+const colourOptions = [
+  { value: 'richard', label: 'Richard' },
+  { value: 'bianca', label: 'Bianca' },
+  { value: 'claire', label: 'Claire' },
+  { value: 'ranon', label: 'Ranon' },
+  { value: 'eric', label: 'Eric' },
+  { value: 'isabel', label: 'Isabel' },
+  { value: 'jane', label: 'Jane' },
+  { value: 'zaid', label: 'Zaid' },
+];
+
+const TagStyle = {
```

Comment:
can we move these to a `Filters.css` file? This [JSS to CSS compiler](https://cssinjs.org/repl/) may be helpful

Reply:
no can do :(